spec/javascripts/utilities/ie11Utils.spec.jsx

```diff
@@ -0,0 +1,14 @@
+import { isBrowserIE11 } from 'utilities/ie11Utils'
+
+describe('isBrowserIE11', () => {
+  it('should return false if browser is not IE11', () => {
```

Comment:
I think the correct statement would be _it should return false if the user agent is other than Trident/7.0_.

Reply:
Updated!